boards/nxp/mr_canhubk3/doc/index.rst

```diff
@@ -296,6 +296,58 @@ For example, to erase and verify flash content:
 
    west flash -r trace32 --startup-args elfFile=build/zephyr/zephyr.elf loadTo=flash eraseFlash=yes verifyFlash=yes
 
+MCUboot
+=======
+
+This board supports app chain-loading using MCUboot.
+
+The command below builds MCUboot and the application together.
+It also generated .hex output files for convenience in flashing.
+
+.. code-block:: console
+
+ $west build -p -b mr_canhubk3/s32k344/mcuboot samples/drivers/flash_shell --sysbuild
```

Comment:
use the proper west build doxygen command for this which will format it nicely

Reply:
Please check now